### PR TITLE
Update Passkey Example for Release

### PR DESCRIPTION
**Pull request by nlordell**
Now that we have released the passkey package, the 4337 example can be updated to use the released packages instead of the development code.

I also change the configuration to make use of the `*-deployments` packages to make sure the example works on the "real" addresses.

I manually ran the example application and was successful with creating and executing user operations:

- [Safe creation userOp](https://sepolia.etherscan.io/tx/0xdcab84d43b4bbd954c13fd991615c2490b0c6e59a058e46764d2875e6dbf4e4f)
- [Transfer userOp](https://sepolia.etherscan.io/tx/0x15210fb522c94907bc38bb99cc9b1020a20e32e19c517f8fed04f607c9c1d666)

### CODE REVIEW

**Review by remedcu — APPROVED** [review]

**Comment by nlordell** [comment]
I want to wait for @mmv08's review before merging (as the "main" code owner of this package).

**Review by mmv08 — APPROVED** [review]

**Pull request merged**
